src/main/java/com/example/demo/controller/dto/request/ArticleCreateRequest.java

```diff
@@ -1,10 +1,10 @@
 package com.example.demo.controller.dto.request;
 
 public record ArticleCreateRequest(
```

Comment:
@JsonNaming 어노테이션을 활용하면 굳이 코드레벨에서 스네이크 케이스의 코드가 돌아다닐 필요가 없습니다.
```suggestion
@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
public record ArticleCreateRequest(
```

참고자료: https://www.baeldung.com/jackson-deserialize-snake-to-camel-case#use-jsonnaming-annotation

Reply:
@JsonNaming이라는 어노테이션을 처음 보는데, 공부해서 코드에 실제 적용해 볼수 있도록 하겠습니다.